cinema/models.py

```diff
@@ -1,9 +1,35 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=63,)
```

Comment:
```suggestion
    first_name = models.CharField(max_length=63)
```
A comma is not required here